src/btouch.cs

```diff
@@ -587,13 +586,12 @@ static int Main2 (string [] args)
 			);
 
 			var g = new Generator (nsManager, public_mode, external, debug, types.ToArray (), strong_dictionaries.ToArray ()){
-				BindThirdPartyLibrary = BindingThirdParty,
```

Comment:
I did this option for a part of a fix that I ended up discarding since I came up with another solution, and figured this part of the fix was still valuable.

The only reason this option was done and not the others, is that I had this actually implemented, and didn't want to spend more time on it.